mypy_django_plugin/transformers/managers.py

```diff
@@ -125,13 +125,19 @@ def _process_dynamic_method(
         variables = []
     args_types = method_type.arg_types[1:]
     if _has_compatible_type_vars(base_that_has_method):
-        ret_type = _replace_type_var(
-            ret_type, base_that_has_method.defn.type_vars[0].fullname, manager_instance.args[0]
-        )
-        args_types = [
-            _replace_type_var(arg_type, base_that_has_method.defn.type_vars[0].fullname, manager_instance.args[0])
-            for arg_type in args_types
-        ]
+        typed_var = manager_instance.args or queryset_info.bases[0].args
+        if (
+            typed_var
+            and isinstance(typed_var[0], Instance)
+            and typed_var[0].type.has_base(
+                fullnames.MODEL_CLASS_FULLNAME,
+            )
```

Comment:
This can fit on one line

```suggestion
            and typed_var[0].type.has_base(fullnames.MODEL_CLASS_FULLNAME)
```

Reply:
I think adding this black setting should fix that automatically https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#c-skip-magic-trailing-comma

I can try a MR for this to see if if it changes a lot of thing in the project ?

Reply:
I think many Python and Black developers are already familiar with the trailing comma behavior. I would rather not deviate from the default.

Just don't add redundant commas when writing new code (unless you want to expand it)

Reply:
In my experience, people were not really aware of this black behaviour, they tend to configure line-length and that's it. 

We recently enabled it at work and it tend to make things more readable. This is personal experience obviously, maybe most people are aware of that.